inc/entityconfig.class.php

```diff
@@ -66,6 +66,10 @@ class PluginFormcreatorEntityconfig extends CommonDBTM {
    const CONFIG_SEARCH_ISSUE_HIDDEN = 0;
    const CONFIG_SEARCH_ISSUE_VISIBLE = 1;
 
+   const CONFIG_UI_FORM_LEGACY = 'legacy';
+   const CONFIG_UI_FORM_FIX = 'fix';
```

Comment:
Some here. Why not something like "uniform_height" ? "V_aligned"

Reply:
done